include/small/small.h

```diff
@@ -236,6 +252,16 @@ small_alloc_check(struct small_alloc *alloc)
 	return slab_cache_check(alloc->cache);
 }
 
+/**
+ * Fill `info' with the information about how `size' bytes would be allocated.
+ * See `struct small_alloc_info' for the description of each field.
+ * Note that this function can return different `info->real_size' for the same
+ * input, depending on the current `small_mempool->used_pool'.
+ */
+void
+small_alloc_info(struct small_alloc *alloc, size_t size,
```

Comment:
I'd pass the allocation pointer to this function as well albeit it'd be ignored. Rationale: if we change the small allocator implementation so that the real allocation size may be different for different allocations of the same size, we wouldn't have to update the API.

Reply:
Done.